app.rb

```diff
@@ -39,6 +39,7 @@ class App < Sinatra::Base
 require 'app/models'
 require 'app/routes'
 require 'app/workers'
+require 'active_support'
```

Comment:
I didn't look into it any further than noticing a jump from 4.1 to 4.2 and solving the exception.

I was surprised it was working pre this as my experience with ActiveSupport is that you have to manually require this.

Would you like me to dig into the cause?